invenio_oauthclient/handlers.py

```diff
@@ -234,20 +235,18 @@ def authorized_signup_handler(resp, remote, *args, **kwargs):
 
         if user is None:
             # Auto sign-up if user not found
-            user = oauth_register(account_info)
-            if user is None:
```

Comment:
@lnielsen do you mean this `if`?
I needed to remove to force to pass from `signup` form in any case.